frontend/src/app/testResults/TestResultInputForm.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import RadioGroup from "../commonComponents/RadioGroup";
+import Button from "../commonComponents/Button";
+import { COVID_RESULTS, TEST_RESULT_DESCRIPTIONS } from "../constants";
+import { v4 as uuidv4 } from "uuid";
+
+interface Props {
+  testResultValue: TestResult | undefined;
+  onChange: (value: TestResult | undefined) => void;
+  onSubmit: () => void;
+}
+
+const TestResultInputForm: React.FC<Props> = ({
+  testResultValue,
+  onSubmit,
+  onChange,
+}) => {
+  const onResultChange = (event: React.FormEvent<HTMLInputElement>) => {
+    const value = (event.target as HTMLInputElement).value as TestResult;
+    onChange(value === testResultValue ? undefined : value);
+  };
+
+  const onResultSubmit = (event: React.FormEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    onSubmit();
+  };
+
+  return (
+    <form className="usa-form">
+      <h4 className="prime-radio__title"> SARS-CoV-2 Results </h4>
+      <React.Fragment>
+        <RadioGroup
+          onChange={onResultChange}
+          buttons={[
+            {
+              value: COVID_RESULTS.POSITIVE,
+              label: `${TEST_RESULT_DESCRIPTIONS.POSITIVE} (+)`,
+            },
+            {
+              value: COVID_RESULTS.NEGATIVE,
+              label: `${TEST_RESULT_DESCRIPTIONS.NEGATIVE} (-)`,
+            },
+            {
+              value: COVID_RESULTS.INCONCLUSIVE,
+              label: `${TEST_RESULT_DESCRIPTIONS.UNDETERMINED}`,
+            },
+          ]}
+          name={`covid-test-result-${uuidv4()}`}
```

Comment:
The queue id value would be ideal for this, since it would create a unique name for each button group.

Reply:
This is a great idea!!! Updated.